.github/workflows/check-merge-labels.yml

```diff
@@ -0,0 +1,33 @@
+name: Check merge labels
+
+on:
+  pull_request:
+    types: [labeled, unlabeled]
+  merge_group:
+permissions: read-all
+
+jobs:
+  label_checks:
+    runs-on: ubuntu-latest
```

Comment:
should this always be `spacetimedb-runner`?

It seemed like that always forced the jobs onto a queue, even for these `run`s that don't require anything fancy.

Reply:
Yeah, generally the `spacetimedb-runner` is going to be much faster compared to the regular github ubuntu-latest runner, but for things like this its fine to just use the github runner.